pkg/meta/annotations.go

```diff
@@ -123,7 +123,16 @@ func IsPauseAnnotationSet(annotations map[string]string) bool {
 // UseVClusterOps returns true if all admin commands should use the vclusterOps
 // library rather than admintools.
 func UseVClusterOps(annotations map[string]string) bool {
-	return lookupBoolAnnotation(annotations, VClusterOpsAnnotation)
+	// UseVClusterOps returns true if the annotation isn't set.
+	defaultValue := true
```

Comment:
@spilchen I got 113 failed unit tests after this change. Is that what you meant by a bunch of test failures you mentioned to me?

Reply:
Yes :)

Reply:
I pushed a change to try and preserve the old unit test behaviour. Let me know what you think of it.